scanner/src/funTest/kotlin/scanners/scancode/ScanCodeScannerFunTest.kt

```diff
@@ -36,13 +36,15 @@ class ScanCodeScannerFunTest : AbstractPathScannerWrapperFunTest(setOf(Expensive
     override val scanner = ScanCode("ScanCode", scannerConfig)
 
     override val expectedFileLicenses = listOf(
-        LicenseFinding("Apache-2.0", TextLocation("LICENSE", 1, 201), 99.94f)
+        LicenseFinding("Apache-2.0", TextLocation("LICENSE", 1, 187), 100.0f),
+        LicenseFinding("Apache-2.0", TextLocation("LICENSE", 191, 201), 100.0f)
```

Comment:
Note, at first I though these required changes might render existing package configurations with license finding curations invalid, but luckily these only depend on the start line, not the end line. So only additional license finding curations might be needed.